packages/react-select/src/components/Select/useSelectStyles.ts

```diff
@@ -0,0 +1,109 @@
+import { makeStyles, mergeClasses } from '@fluentui/react-make-styles';
+import type { SelectState } from './Select.types';
+import type { Theme } from '@fluentui/react-theme';
+
+const contentSizes = {
+  // TODO: borrowed this from Input, should be in the theme somewhere?
```

Comment:
We have proposed sharing common style mixins, but there's no conclusion on that unfortunately :(

If these styles are the same as what is in `Text` then we should reevaluate this, because we shouldn't copy+paste styles

Reply:
I agree, going to leave this convo to the styling PR to not overload this one (and removing all specific styles from this PR, since I was rewriting them anyway)